diff --git a/configure.ac b/configure.ac
index 8a7eb03f..2201f6da 100644
--- a/configure.ac
+++ b/configure.ac
@@ -230,7 +230,7 @@ AS_IF([test "x$enable_guile" != "xno"],[
     GUILE_FLAGS
     AC_DEFINE_UNQUOTED([GUILE_BINARY],"$GUILE",[guile binary])
     AC_DEFINE(BUILD_GUILE,[1], [Do we support Guile?])
-    AC_SUBST(GUILE_SNARF, [guile-snarf])
+    AC_SUBST(GUILE_SNARF, [guile-snarf2.2])
     guile_version=$($PKG_CONFIG guile-2.2 --modversion)
   ])
 ])
diff --git a/guile/Makefile.am b/guile/Makefile.am
index 319b0798..10c3dbc2 100644
--- a/guile/Makefile.am
+++ b/guile/Makefile.am
@@ -68,7 +68,7 @@ SUFFIXES = .x .doc
 
 # FIXME: GUILE_SITEDIR would be better, but that
 # breaks 'make distcheck'
-scmdir=${prefix}/share/guile/site/2.2/
+scmdir=${prefix}/share/guile/2.2/
 scm_DATA=mu.scm
 
 EXTRA_DIST=$(scm_DATA)
diff --git a/guile/mu-guile.texi b/guile/mu-guile.texi
index c52213ec..7cfd02a0 100644
--- a/guile/mu-guile.texi
+++ b/guile/mu-guile.texi
@@ -176,12 +176,12 @@ If necessary, you can add the @t{%load-path} by adding to your
 @file{~/.guile}:
 
 @lisp
-(set! %load-path (cons "/usr/local/share/guile/site/2.0" %load-path))
+(set! %load-path (cons "/usr/local/share/guile/2.0" %load-path))
 @end lisp
 
 Or, alternatively, you can set @t{GUILE_LOAD_PATH}:
 @example
-export GUILE_LOAD_PATH=/usr/local/share/guile/site/2.0
+export GUILE_LOAD_PATH=/usr/local/share/guile/2.0
 @end example
 
 In both cases the directory should be the directory that contains the
diff --git a/guile/mu/Makefile.am b/guile/mu/Makefile.am
index f531822c..afed7e2d 100644
--- a/guile/mu/Makefile.am
+++ b/guile/mu/Makefile.am
@@ -18,7 +18,7 @@ include $(top_srcdir)/gtest.mk
 
 # FIXME: GUILE_SITEDIR would be better, but that
 # breaks 'make distcheck'
-scmdir=${prefix}/share/guile/site/2.2/mu/
+scmdir=${prefix}/share/guile/2.2/mu/
 
 scm_DATA=		\
 	stats.scm	\
diff --git a/www/mu-guile.md b/www/mu-guile.md
index 168afc30..a8f16ce7 100644
--- a/www/mu-guile.md
+++ b/www/mu-guile.md
@@ -32,7 +32,7 @@ the `GUILE_LOAD_PATH` to the directory with the installed `mu.scm`,
 e.g.
 
 ``` sh
-   export GUILE_LOAD_PATH="/usr/local/share/guile/site/2.0"
+   export GUILE_LOAD_PATH="/usr/local/share/guile/2.0"
 ```
 
 (you need to adapt this if you installed `mu` in some non-standard place; but
diff --git a/www/mu-guile.org b/www/mu-guile.org
index de6ad7db..84f5af8e 100644
--- a/www/mu-guile.org
+++ b/www/mu-guile.org
@@ -24,7 +24,7 @@
    ~GUILE_LOAD_PATH~ to the directory with the installed ~mu.scm~, e.g.
 
 #+begin_src sh
-   export GUILE_LOAD_PATH="/usr/local/share/guile/site/2.0"
+   export GUILE_LOAD_PATH="/usr/local/share/guile/2.2"
 #+end_src
 
    (you need to adapt this if you installed =mu= in some non-standard place; but
